src/metaopt/io/database/mongodb.py

```diff
@@ -0,0 +1,137 @@
+import json
+import bson
+import pymongo
+
+from pymongo import MongoClient
+from datetime import datetime
+from abstractdb import AbstractDB
+
+try:
+    from StringIO import StringIO
```

Comment:
In which case StringIO is not available?

Reply:
In Python 3.0 `StringIO` and `cStringIO` modules are gone. See https://stackoverflow.com/questions/11914472/stringio-in-python3

Reply:
Ah! You are speaking to an ignorant of python3 indeed.

Reply:
Gotcha :+1:

Reply:
http://python-future.org/compatible_idioms.html

You can use `from io import StringIO`.

In general, these sites are good references along with the `six` module:

http://python-future.org/compatible_idioms.html
https://docs.python.org/3/howto/pyporting.html
https://pythonhosted.org/six/